lib/text_confirmation.rb

```diff
@@ -0,0 +1,34 @@
+require 'dotenv'
+require 'twilio-ruby'
+
+class TextConfirmation
```

Comment:
It feels like TextConfirmation has two responsibilities: sending texts and composing texts about takeaway orders.  Could these responsibilities be split up?